index.js

```diff
@@ -285,15 +286,15 @@ class Collection extends BaseAPI {
    * @param {number} options.depth The recursion depth to fetch collections. (default: 1)
    * @param {Object} options.storyLimits The limit of stories to fetch by collection template. This defaults to unlimited for templates that are not specified. (ex: {"FourColGrid": 12}) (default: {}).
    * @param {number} options.defaultNestedLimit The default limit of stories to fetch by each collection. (default: 40)
-   * @param {Object} options.nestedCollectionLimit The number of stories or collection to fetch from each nested collection. (Ex: nestedCollectionLimit: {ThreeColGrid: [2, 3, 4]}). 
+   * @param {Object} options.nestedCollectionLimit The number of stories or collection to fetch from each nested collection. (Ex: nestedCollectionLimit: {ThreeColGrid: [2, 3, 4]}).
    eg:
-    - Home `(Level 1)`
-     - Sports Row `(Level 2)` `(template- ThreeColGrid)`
-      - Cricket `(Level 3)`
-      - Football `(Level 3)`
-      - Tennis `(Level 3)`
```

Comment:
I think this indentation was applied on purpose to indicate the structure of represented data.

Reply:
Fixed